src/main/java/com/example/zawody/controller/AppController.java

```diff
@@ -0,0 +1,34 @@
+package com.example.zawody.controller;
+
+
+import com.example.zawody.model.DTO;
+import com.example.zawody.model.Person;
+import com.example.zawody.repositories.PersonRepository;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+
+@RestController
+public class AppController {
+
+    PersonRepository personRepository;
+
+    public AppController(PersonRepository personRepository) {
+        this.personRepository = personRepository;
+    }
+
+    @RequestMapping(value = "/posted",method = RequestMethod.POST,consumes = MediaType.APPLICATION_JSON_VALUE)
+    @CrossOrigin("http://localhost:8080")
+    public ResponseEntity<Void>save(@RequestBody Person person) {
+
+        personRepository.save(person);
```

Comment:
Proper architecture would look like: `ResultController --calls-->  ResultService.addResult(resultDto) --calls--> CompetitionRepository.save(competition) AND PersonRepository.save(person)`

You should also be aware of the possible transactional issues. What if Competition node saves correctly, and person save fails? Read about how to handle such situations in neo4j